lib/vdsm/tool/configurators/passwd.py

```diff
@@ -95,7 +95,7 @@ def removeConf():
 def configure_libvirt_sasl():
     with io.open(_SASL2_CONF, 'w', encoding='utf8') as f:
         f.writelines([u'## start vdsm-4.20.0 configuration\n',
```

Comment:
Using 4.20.0 does not make sense now. If this configuration was added in vdsm-4.5.x we should use right value.

Note this string may be used by other parts to check if the file is configured for vdsm. It looks like this implementation is broken (broken your patch), hard coding vdsm version that is not relevant for ages.

Please check how other configurators handle this.

Reply:
The version strings added by the configurators were always "arbitrary", but we tried to keep them in sync with actual vdsm versions when changes were introduced. Given that we're actually changing the config in this patch it would be nice to bump the version to `4.50.0` too.

Reply:
Done